instat/dlgRugPlot.vb

```diff
@@ -125,7 +102,28 @@ Public Class dlgRugPlot
         clsBaseOperator.SetAssignTo("last_graph", strTempDataframe:=ucrRugPlotSelector.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempGraph:="last_graph")
         ucrBase.clsRsyntax.SetBaseROperator(clsBaseOperator)
         bResetSubdialog = True
+    End Sub
 
+    Public Sub SetRCodeForControls(bReset As Boolean)

```

Comment:
Moved Initialise and SetDefaults to be at the start of this class
Made this a Public sub()